requirements.txt

```diff
@@ -3,3 +3,5 @@ logfury>=1.0.1,<2.0.0
 requests>=2.9.1,<3.0.0
 tqdm>=4.5.0,<5.0.0
 typing-extensions>=4.7.1; python_version < '3.12'
+setuptools>=60.0
+packaging>=23.0
```

Comment:
packaging 23 is especially recent, if we relax this (for example, using >=21) it would be safer - it seems we are not using any new feature of packaging

pip only "recently" started properly resolving conflicting dependencies, people with python 3.7 or even 3.8 may still have that version

Reply:
Between related danger and fact it does silence deprecation warning - it would be nice to include it in the changelog